meta-lmp-base/conf/layer.conf

```diff
@@ -19,7 +19,7 @@ LAYERDEPENDS_meta-lmp-base = " \
     sota \
     security \
 "
-LAYERSERIES_COMPAT_meta-lmp-base = "kirkstone"
+LAYERSERIES_COMPAT_meta-lmp-base = "langdale"
```

Comment:
maybe we can go straight to scarthgap?

Reply:
Same reason of the previous comment, after kirkstone version came langdale.